hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/SubsystemRemove.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
```

Comment:
since this file is no longer changed, can we revert it back (remove this change to the license header). One less file to be considered "changed" for this PR.